app/2.0.x/proxy.md

```diff
@@ -11,11 +11,13 @@ Kong exposes several interfaces which can be tweaked by two configuration
 properties:
 
 - `proxy_listen`, which defines a list of addresses/ports on which Kong will
-  accept **public traffic** from clients and proxy it to your upstream services
-  (`8000` by default).
+  accept **public HTTP (gRPC, WebSocket, etc) traffic** from clients and proxy
+  it to your upstream services (`8000` by default).
 - `admin_listen`, which also defines a list of addresses and ports, but those
   should be restricted to only be accessed by administrators, as they expose
   Kong's configuration capabilities: the **Admin API** (`8001` by default).
+- `stream_listen`, which is similar to `proxy_listen` but for Layer 4 (TCP, TLS)
+  generic proxy. This is turned off by default.
```

Comment:
doesn't this make _three_ configuration properties? (the paragraph above says "tweaked by two configuration
properties")

Reply:
Thanks, fixed.